nilearn/plotting/html_surface.py

```diff
@@ -21,20 +21,39 @@ class SurfaceView(HTMLDocument):
 
 
 def _get_vertexcolor(surf_map, cmap, norm,
-                     absolute_threshold=None, bg_map=None):
+                     absolute_threshold=None, bg_map=None,
+                     bg_on_data=None, scale_bg_map=None, darkness=None):
```

Comment:
I guess we will be able to add these parameters to `view_surf` and `view_img_on_surf`? (probably in another PR)

Reply:
I did it in this PR :slightly_smiling_face: